Makefile

```diff
@@ -469,17 +465,6 @@ qe-graphql:
 qe-dmmf:
 	cargo run --bin query-engine -- cli dmmf > dmmf.json
 
-push-schema:
-	cargo run --bin test-cli -- schema-push $(DEV_SCHEMA_FILE) --force
-
-qe-dev-chinook-sqlite:
-	cp $(SCHEMA_EXAMPLES_PATH)/chinook_sqlite.prisma $(DEV_SCHEMA_FILE)
-
-qe-dev-chinook-postgres10: start-postgres10
-	cp $(SCHEMA_EXAMPLES_PATH)/chinook_postgres10.prisma $(DEV_SCHEMA_FILE)
-	sleep 5
-	make push-schema
```

Comment:
This command hasn't been used in years, and was making use of `make push-schema`.
Similar rationale applies to `qe-dev-chinook-sqlite`.